client/src/styles/datetime.css

```diff
@@ -0,0 +1,81 @@
+.react-datepicker__input-container input {
```

Comment:
The UI works well on desktop but not on mobile, e.g. iPhone X:
![Screen Shot 2020-06-01 at 18 25 03](https://user-images.githubusercontent.com/30998659/83401387-466e1f00-a437-11ea-8c8a-bf40e700bd93.png)
I suggest using [css media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/@media) and adjust the padding to fix it. Refer [here](https://github.com/wasedatime/wasedatime-web/blob/master/client/src/js/styled-components/utils.js) for the three defined screen sizes in WasedaTime.
Next time don't forget to check the display on mobile devices~ You can do so using the developer tools in Chrome.

Reply:
Got it! I will fix this before Thursday.

Reply:
I've added the size for datepicker on mobile mode. Please check!